storage/preferences/src/main/java/de/danoeh/antennapod/storage/preferences/UserPreferences.java

```diff
@@ -227,6 +231,57 @@ public static boolean showSkipOnCompactNotification() {
         return showButtonOnCompactNotification(NOTIFICATION_BUTTON_SKIP);
     }
 
+    public static boolean showNextChapterOnCompactNotification() {
+        return showButtonOnCompactNotification(NOTIFICATION_BUTTON_NEXT_CHAPTER);
+    }
+
+    public static List<Integer> getFullNotificationButtons() {
+        String[] buttons = TextUtils.split(
+            prefs.getString(PREF_FULL_NOTIFICATION_BUTTONS,
+                NOTIFICATION_BUTTON_REWIND + ","
+                    + NOTIFICATION_BUTTON_FAST_FORWARD + ","
+                    + NOTIFICATION_BUTTON_SKIP),
+            ",");
+
+        List<Integer> notificationButtons = new ArrayList<>();
+        for (String button : buttons) {
+            notificationButtons.add(Integer.parseInt(button));
+        }
+        return notificationButtons;
+    }
+
+    /**
+     * Helper function to return whether the specified button should be shown on full
+     * notifications.
+     *
+     * @param buttonId Either NOTIFICATION_BUTTON_REWIND, NOTIFICATION_BUTTON_FAST_FORWARD,
+     *                 NOTIFICATION_BUTTON_SKIP or NOTIFICATION_BUTTON_PLAYBACK_SPEED.
+     * @return {@code true} if button should be shown, {@code false}  otherwise
+     */
+    private static boolean showButtonOnFullNotification(int buttonId) {
+        return getFullNotificationButtons().contains(buttonId);
+    }
+
+    public static boolean showRewindOnFullNotification() {
+        return showButtonOnFullNotification(NOTIFICATION_BUTTON_REWIND);
+    }
+
+    public static boolean showFastForwardOnFullNotification() {
+        return showButtonOnFullNotification(NOTIFICATION_BUTTON_FAST_FORWARD);
+    }
+
+    public static boolean showSkipOnFullNotification() {
+        return showButtonOnFullNotification(NOTIFICATION_BUTTON_SKIP);
+    }
+
+    public static boolean showNextChapterOnFullNotification() {
+        return showButtonOnFullNotification(NOTIFICATION_BUTTON_SKIP);
```

Comment:
@matejdro, you changed it above but didn't change it here. This should also be `NOTIFICATION_BUTTON_NEXT_CHAPTER`.

Reply:
:man_facepalming:  thanks